content/en/docs/File Format/implementationstatus.md

```diff
@@ -109,14 +111,14 @@ Implementations:
 
 ### High level data APIs for Parquet feature usage
 
-| Format                                       | C++   | Java   | Go    | Rust  |
-| -------------------------------------------- | ----- | ------ | ----- | ----- |
-| External column data (1)                     |       |        |       |       |
-| Row group "Sorting column" metadata (2)      |       |        |       |       |
-| Row group pruning using statistics           |       |        |       |       |
-| Reading select columns only                  |       |        |       |       |
-| Page pruning using statistics                |       |        |       |       |
-| Page pruning using bloom filter              |       |        |       |       |
+| Format                                       | C++   | Java   | Go    | Rust  | CUDA  |
+| -------------------------------------------- | ----- | ------ | ----- | ----- | ----- |
+| External column data (1)                     |       |        |       |       |  (W)  |
+| Row group "Sorting column" metadata (2)      |       |        |       |       |  (W)  |
+| Row group pruning using statistics           |       |        |       |       |  ✅   |
+| Row group pruning using bloom filter         |       |        |       |       |  ✅   |
```

Comment:
Please correct me if I am wrong but I believe the bloom filters are used to prune row groups instead of pages.